src/utils/rfqm_db_utils.ts

```diff
@@ -22,6 +22,21 @@ export interface RfqmJobOpts {
     metadata?: object;
 }
 
+export interface RfqmTransactionSubmissionOpts {
```

Comment:
why not extend `TxOpts`?

Reply:
this is for the db, not the web3 call

Reply:
Maybe put `Storage` or something like that somewhere in the name of the storage types so that's clear